sky/data/storage.py

```diff
@@ -1246,8 +1307,45 @@ def mount_command(self, mount_path: str) -> str:
                      f'--stat-cache-ttl {self._STAT_CACHE_TTL} '
                      f'--type-cache-ttl {self._TYPE_CACHE_TTL} '
                      f'{self.bucket.name} {mount_path}')
-        return mounting_utils.get_mounting_command(mount_path, install_cmd,
-                                                   mount_cmd)
+        return mounting_utils.get_mounting_command(StorageMode.MOUNT,
+                                                   mount_path, mount_cmd,
+                                                   install_cmd)
+
+    def csync_command(self,
```

Comment:
There is a lot of shared code in `S3Store. csync_command` and `GcsStore.csync_command`. Can we reuse some of them?

Reply:
Great catch. Removed `csync_command` from each `S3Store` and `GcsStore` class and implemented `csync_command` under `AbstractStore` class.